### PR TITLE
Linkage Monitor

### LINKED ISSUE #718: Kokoro build config Linkage Monitor
As discussed with @chingor13 , I'm going to commit cloud-http-java-client Kokoro build config and shell script to run Linkage Monitor as part of presubmit.

This issue is a part of https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/718

CC: @chingor13

### PR DESCRIPTION
**Pull request by suztomo**
@chingor13 

[Linkage Monitor](https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/linkage-monitor) notifies new linkage errors as presubmit check.

Fixes #718 

- [x] Tests pass
- [ ] Appropriate docs were updated (if necessary)

The presubmit job will not run until Kokoro job config is created (cl/253256348).

### CODE REVIEW

File: .kokoro/linkage-monitor.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```
Comment (chingor13) [inline .kokoro/linkage-monitor.sh]: Looks like this file needs to be made executable

Comment (suztomo) [reply .kokoro/linkage-monitor.sh]: Fixed!
https://github.com/googleapis/google-http-java-client/pull/719/commits/ba63ad29b4efdfa7762b1986d312929633b990f0

File: .kokoro/linkage-monitor.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# Installs snapshot artifacts locally. Linkage Monitor uses them.
+mvn install -DskipTests=true -B -V
+
+# Kokoro job cloud-opensource-java/ubuntu/linkage-monitor-gcs creates this JAR
+JAR=linkage-monitor-latest-all-deps.jar
+curl -v -O "https://storage.googleapis.com/cloud-opensource-java-linkage-monitor/${JAR}"
+
+# Fails if there's new linkage errors compared with baseline
+java -jar $JAR com.google.cloud:libraries-bom
```
Comment (chingor13) [inline .kokoro/linkage-monitor.sh]: Looks like the libraries-bom artifact needs a version

Comment (suztomo) [reply .kokoro/linkage-monitor.sh]: Great. The error message is expected (sorry the tool is not complete yet). Good for now. Let's keep this PR open.

Comment (suztomo) [reply .kokoro/linkage-monitor.sh]: Now it works!

**Pull request merged**